validator/sawtooth_validator/journal/publisher.py

```diff
@@ -194,6 +194,9 @@ def finalize_block(self, identity_signing_key, pending_batches):
         :param pending_batches: list to receive any batches that were
         submitted to add to the block but were not validated before this
         call.
+        :return: None if the block failed to finalize or the Block that was
```

Comment:
Unclear if the _"or"_ refers to under what conditions `None` will be returned, or if it refers to the possible return values. Suggested rewording:

_The generated Block, or None if Block failed to finalize._